components/nifosg/controller.cpp

```diff
@@ -85,7 +85,7 @@ KeyframeController::KeyframeController(const KeyframeController &copy, const osg
 }
 
 KeyframeController::KeyframeController(const Nif::NiKeyframeData *data)
-    : mRotations(data->mRotations)
+    : mRotations(data->mRotations, osg::Quat())
```

Comment:
I had a compile error in GCC 5.2 that pointed me at this line for some reason.  Maybe I misunderstood.